internal/integration/harness.go

```diff
@@ -303,7 +305,9 @@ func run(t *testing.T, actionsOrOptions ...ActionOrOption) {
 			err = ftlexec.Command(ctx, log.Debug, rootDir, "just", "build-without-frontend", "ftl").RunBuffered(ctx)
 			assert.NoError(t, err)
 		}
-		if opts.requireJava || slices.Contains(opts.languages, "java") || slices.Contains(opts.languages, "kotlin") {
+		if os.Getenv("USE_RELEASE_BINARIES") == "1" {
+			Infof("Using pre-built JARs")
```

Comment:
It doesn't look like you are building and uploading the JVM artifacts as part of the setup? I think this may be testing the JVM binaries from main, rather than ones built for this PR.

Reply:
I don't think this was ready for merging yet?